testinfra/modules/file.py

```diff
@@ -169,6 +169,18 @@ def size(self):
         """Return size of file in bytes"""
         raise NotImplementedError
 
+    @property
+    def list_dir(self):
```

Comment:
Maybe `listdir()` is more straightforward for python people (like `os.listdir()`)

Reply:
I agree resemblance to Python standard library could be a plus. I followed the convention from current module ([is_file](https://github.com/philpep/testinfra/blob/392a3b8f5e958a152b8b34f1fb7a907b5c67d6cc/testinfra/modules/file.py#L42), [is_directory](https://github.com/philpep/testinfra/blob/392a3b8f5e958a152b8b34f1fb7a907b5c67d6cc/testinfra/modules/file.py#L46), etc.), all functions are snake_cased and none named as attachedwords. Do you still prefer `list_dir` to be renamed to `listdir`?

Reply:
I'd prefer listdir (not a strong opinion through)